openlibrary/macros/SearchResultsWork.html

```diff
@@ -88,29 +86,36 @@ <h3 itemprop="name" class="booktitle">
                 }
                 for a in authors
               ]
-            $:macros.BookByline(author_data, limit=max_rendered_authors, overflow_url=work_edition_url, attrs='class="results"')
```

Comment:
by removing the class="results" in this BookByline call, I removed the styling from [.results in buttonsAndLinks.less](https://github.com/internetarchive/openlibrary/blob/f4222d30a8805fe1fe820acfed6b74f7d0f90b0b/static/css/components/buttonsAndLinks.less#L44) that was overriding the native blue link color.